templates/server/puppetserver/conf.d/auth.conf.erb

```diff
@@ -77,6 +77,20 @@ authorization: {
             name: "puppetlabs csr"
         },
 <%- if @server_ca -%>
+<%- if versioncmp(@real_puppetserver_version, '8.0.0') >= 0 -%>
```

Comment:
Looking at 689d12150e12017f0675e46204c39aef06ea9b45 it needs to be this to call the function in ERB
```suggestion
<%- if scope.function_versioncmp([@real_puppetserver_version, '8.0.0']) >= 0 -%>
```